tests/e2e/accounts/multisig/account_test.go

```diff
@@ -0,0 +1,307 @@
+	accountsv1 "cosmossdk.io/x/accounts/v1"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func TestE2ETestSuite(t *testing.T) {
+	suite.Run(t, NewE2ETestSuite())
+}
+
+// TestFullMultisig creates a multisig account with 1 member, sends a tx, votes and executes it
+// then adds 2 more members and changes the config to require 2/3 majority (also through a proposal).
+// Finally it creates a proposal that won't pass.
+func (s *E2ETestSuite) TestFullMultisig() {
+	t := s.T()
+	app := setupApp(t)
+	currentTime := time.Now()
+	ctx := sdk.NewContext(app.CommitMultiStore(), false, app.Logger()).WithHeaderInfo(header.Info{
+		Time: currentTime,
+	})
+	member0, err := app.AuthKeeper.AddressCodec().BytesToString(members[0])
+	require.NoError(t, err)
+	member1, err := app.AuthKeeper.AddressCodec().BytesToString(members[1])
+	require.NoError(t, err)
+	member2, err := app.AuthKeeper.AddressCodec().BytesToString(members[2])
+	require.NoError(t, err)
+
+	s.fundAccount(app, ctx, members[0], sdk.Coins{sdk.NewCoin("stake", math.NewInt(1000000))})
+	randAcc := sdk.AccAddress(secp256k1.GenPrivKey().PubKey().Address())
+
+	_, accountAddr, err := app.AccountsKeeper.Init(ctx, multisigaccount.MULTISIG_ACCOUNT, members[0], &v1.MsgInit{
+		Members: []*v1.Member{{Address: member0, Weight: 100}},
+		Config: &v1.Config{
+			Threshold:      100,
+			Quorum:         100,
+			VotingPeriod:   120,
+			Revote:         false,
+			EarlyExecution: true,
+		},
+	}, sdk.Coins{sdk.NewCoin("stake", math.NewInt(1000))})
+	require.NoError(t, err)
+
+	addr, err := app.AuthKeeper.AddressCodec().BytesToString(randAcc)
+	require.NoError(t, err)
+
+	accAddrStr := sdk.AccAddress(accountAddr).String()
+
+	balance := app.BankKeeper.GetBalance(ctx, randAcc, "stake")
+	require.True(t, balance.Amount.Equal(math.NewInt(0)))
+
+	// do a simple bank send
+	msg := &bankv1beta1.MsgSend{
+		FromAddress: accAddrStr,
+		ToAddress:   addr,
+		Amount: []*basev1beta1.Coin{
+			{
+				Denom:  "stake",
+				Amount: "100",
+			},
+		},
+	}
+	anyMsg, err := codectypes.NewAnyWithValue(msg)
+	require.NoError(t, err)
+
+	propReq := &v1.MsgCreateProposal{
+		Proposal: &v1.Proposal{
+			Title:    "test",
+			Summary:  "test",
+			Messages: []*codectypes.Any{anyMsg},
+		},
+	}
+	err = s.executeTx(ctx, propReq, app, accountAddr, members[0])
+	require.NoError(t, err)
+
+	// now we vote for it
+	voteReq := &v1.MsgVote{
+		ProposalId: 0,
+		Vote:       v1.VoteOption_VOTE_OPTION_YES,
+	}
+
+	err = s.executeTx(ctx, voteReq, app, accountAddr, members[0])
+	require.NoError(t, err)
+
+	// now we execute it
+	execReq := &v1.MsgExecuteProposal{
+		ProposalId: 0,
+	}
+
+	err = s.executeTx(ctx, execReq, app, accountAddr, members[0])
+	require.NoError(t, err)
+
+	for _, v := range ctx.EventManager().Events() {
+		if v.Type == "proposal_tally" {
+			status, found := v.GetAttribute("status")
```

Comment:
I assume `proposal_tally` should be in the list of events but you don't ensure that here and below